generate_side_by_side.py

```diff
@@ -559,6 +690,7 @@ def _search_for_paths(rev_ids):
                 ingest_continue=False,
             )
             base_paths = _search_for_paths([base_revision_id])
+            base_vismet = get_perfherder_data(base_revision_id, str(output), vismet_platform, test_no_e10s)
```

Comment:
You could replace this with the following after some modifications: `_search_for_paths([base_revision_id], artifact="perfherder-data")`